django/pagai/api/views.py

```diff
@@ -40,28 +40,28 @@ def post(self, request, owner):
         return Response(db_schema, status=status.HTTP_200_OK)
 
 
-class ExploreView(views.APIView):
-    def get_serializer_class(self):
-        return ExplorationSerializer
+@extend_schema(
+    request=serializers.ExplorationRequestSerializer, responses={"201": serializers.ExplorationResponseSerializer}
```

Comment:
Why is this a 201?

Reply:
I don't have a preference between a 200 or a 201 but since we use the create method, I thought it was appropriate..

Reply:
Yeah but we don't actually create something as you said in another comment